### PR TITLE
fix: add config to target the good ws kcp

### PR DESCRIPTION
**Pull request by aureamunoz**

### CODE REVIEW

File: scenario-2.sh [hunk 1]
```diff
@@ -39,25 +39,27 @@ KUBECONFIG=_tmp/.kcp/admin.kubeconfig k apply -f ./k8s
 
 note "Moving to the root:${KCP_WORKSPACE} workspace"
 note ">> k kcp ws use root:${KCP_WORKSPACE}"
-k kcp ws use root:${KCP_WORKSPACE}
+KUBECONFIG=_tmp/.kcp/admin.kubeconfig k kcp ws use root:${KCP_WORKSPACE}
```
Comment (cmoulliard) [inline scenario-2.sh]: Can you replace the hard coded path `_tmp` with the var `${TEMP_DIR}` ?

Comment (aureamunoz) [reply scenario-2.sh]: oups! yes

Comment (aureamunoz) [reply scenario-2.sh]: done

**Comment by cmoulliard** [comment]
LGTM

**Pull request merged**